rotkehlchen/chain/ethereum/node_inquirer.py

```diff
@@ -96,6 +96,18 @@ def __init__(
         self.etherscan = cast(EthereumEtherscan, self.etherscan)
         self.ens_reverse_records = self.contracts.contract('ENS_REVERSE_RECORDS')
 
+    def ens_avatar_lookup(self, ens_name: str) -> Optional[str]:
+        """
+        Checks whether given ens name has an ens avatar set.
+        If there is avatar set -- returns the url of the avatar.
+        """
```

Comment:
Thanks. Will add a note about the remote error, however I don't think we should return `None` in case of a remote error :thinking:

If we do it, then a remote error will be equal to non-existence of an avatar, which is not correct.

Reply:
You could log an error...
However, not handling `RemoteError` here is synonymous to returning `None`.

Or maybe handle the `RemoteError` where the method is used.

Reply:
Now is handled